.ci.yaml

```diff
@@ -3893,7 +3893,7 @@ targets:
       - bin/**
       - .ci.yaml
 
-  - name: Windows tool_integration_tests_1_5
+  - name: Windows tool_integration_tests_1_6
```

Comment:
This unfortunately still shows some of the pain with having to roll into the LUCI configs. We need to duplicate these builders with bringup: true, then land a separate commit to remove the label

Once we move flutter/infra to a generic builder we can get around this

Reply:
You mean leave `tool_integration_tests_1_5` as blocking and add `tool_integration_tests_1_6` as bringup? and then after it rolls to the builder config, delete the old ones and mark the new ones as blocking?

Reply:
Yes. Post-submit will seem like there are duplicates, which is fine.

Reply:
Filed https://github.com/flutter/flutter/issues/96544 to track the follow-up PR